ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -1024,7 +1024,16 @@ public void AddApplicableTradeTax(decimal basisAmount, decimal percent, TaxTypes
         /// <param name="allowanceChargeBasisAmount"></param>
         /// <param name="exemptionReasonCode"></param>
         /// <param name="exemptionReason"></param>        
-        public void AddApplicableTradeTax(decimal basisAmount, decimal percent, decimal taxAmount, TaxTypes typeCode, TaxCategoryCodes? categoryCode = null, decimal? allowanceChargeBasisAmount = null, TaxExemptionReasonCodes? exemptionReasonCode = null, string exemptionReason = null)
+        /// <param name="lineTotalBasisAmount">Basis amount of the line total</param>
+        public void AddApplicableTradeTax(decimal basisAmount,
+            decimal percent,
+            decimal taxAmount,
+            TaxTypes typeCode,
+            TaxCategoryCodes? categoryCode = null,
+            decimal? allowanceChargeBasisAmount = null,
+            TaxExemptionReasonCodes? exemptionReasonCode = null,
+            string exemptionReason = null,
+            decimal? lineTotalBasisAmount = null)
```

Comment:
add documentation for the new parameter. You can use the same description as the property description in Tax.LineTotalBasisAmount